docker/memcached.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 # `/sbin/setuser memcache` runs the given command as the user `memcache`.
 # If you omit that part, the command will be run as root.
-exec /sbin/setuser memcache /usr/bin/memcached >>/var/log/memcached.log 2>&1
+
+#read conf file
+options=$(cat /etc/memcached.conf | grep -v '#' | grep -v '^$' | tr '\n' ' ')
+
+exec /usr/bin/memcached $options >>/var/log/memcached.log 2>&1
```

Comment:
This should be run as `exec /sbin/setuser memcache /usr/bin/memcached` so that `memcached` does not run with root privileges. Looks good otherwise.

Reply:
Makes sense, thanks for explanation.